pandas/tests/plotting/common.py

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env python3
```

Comment:
do we need this shebang?

Reply:
I meant to do this as follow up PR, trying to have each PR for specific thing.